### PR TITLE
Cleanup super usage in toolkit agnostic code

### PR DESCRIPTION
**Pull request by rahulporuri**
This PR updates `super` usage in a semi-automated fashion. We used regex-based search and replace to update the `super` usage. Note that there was one unnnecessary redefinition of `__init__` which got removed in this PR.

See similar PR #944, #945 and https://github.com/enthought/traits/pull/1280

### CODE REVIEW

File: pyface/dock/dock_sizer.py [hunk 1]
```diff
@@ -3830,9 +3830,6 @@ class DockInfo(HasPrivateTraits):
     # Dock Control:
     control = Instance(DockItem)
 
-    def __init__(self, **kw):
-        super(DockInfo, self).__init__(**kw)
```
Comment (rahulporuri) [inline pyface/dock/dock_sizer.py:3834]: this is an unnecessary redefinition of `__init__` which we removed

File: pyface/dock/dock_sizer.py [hunk 2]
```diff
@@ -432,7 +432,7 @@ class DockItem(HasPrivateTraits):
     # ---------------------------------------------------------------------------
 
     def __init__(self, **kw):
-        super(DockItem, self).__init__(**kw)
+        super().__init__(**kw)
```
Comment (aaronayres35) [inline pyface/dock/dock_sizer.py:435]: can this just be removed entirely?

**Review by aaronayres35 — APPROVED** [review]
LGTM

just one additional unnecessary redefinition of `__init__` that could be removed

File: pyface/dock/dock_sizer.py (hunk 2 above)
Comment (rahulporuri) [reply pyface/dock/dock_sizer.py]: removed

**Pull request merged**